src/main/java/com/flippingutilities/utilities/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.flippingutilities.utilities;
+
+public class Constants {
```

Comment:
Could set the 1% value in here as well
Perhaps the non-taxed items as well? Although these are very few atm so it might not be worth bothering.